client/agenda/AgendaQuickAccess.vue

```diff
@@ -206,8 +206,6 @@ function scrollToDay (dayId, ev) {
 }
 
 function scrollToNow (ev) {
-  ev.preventDefault()
```

Comment:
Removed so that when clicking 'Now' (`<a class="nav-link" href="#now"><i class="bi bi-arrow-right-short d-none d-xxl-inline me-2"></i><span>Now</span></a>` ... ![Screenshot from 2024-08-01 10-52-50](https://github.com/user-attachments/assets/9cff8ec8-7969-4261-8f62-91c6b8170a6c) )

...it will update the URL with `#now` so that future reloads will have a hashstate that expresses 'Now'